scripts/lint.sh

```diff
@@ -28,5 +28,5 @@ inv -e install-tools
 inv -e deps
 
 # Run go linters
-inv -e linter.go --cpus 3
```

Comment:
Why 30min? Would it be worth to go for a bigger increase? Like 50 minutes or even 1h?

Reply:
Silly question, could we also increase the `--cpus` (if the infra is adapted)?

Reply:
`--cpus` should target the amount of cores available on the runner, otherwise it decreases performance, we could read the number of cores from .

It would require running `system_profiler SPHardwareDataType` or a python script to get the number of cpu and then use it here to optimise this.

Not sure it is safe to do it in this PR though

Reply:
The number of CPUs made available to GitHub Actions runners is publicly known. It is 3 for the `macos-11` runners we use (see [runner docs](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories)).

Reply:
About the time last evening I thought that the max time on a normal runner for a golangci-lint package is 14 minutes, but I cannot find the info today, looking for it

Reply:
Could not find why I said 30 min, let's move to 1h